packages/ra-ui-materialui/src/field/BooleanField.stories.tsx

```diff
@@ -22,3 +22,32 @@ export const Basic = (props: BooleanFieldProps) => {
 export const NoFalseIcon = () => <Basic FalseIcon={null} />;
 
 export const NoTrueIcon = () => <Basic TrueIcon={null} />;
+
+type Post = {
```

Comment:
That's not a record, it requires an `id`. I'm surprised that TypeScript doesn't warn you about this one.

Reply:
Because our fields don't require a RaRecord anymore